opslevel/team.go

```diff
@@ -21,11 +21,12 @@ type ContactInput struct {
 }
 
 type Team struct {
-	Id               graphql.String
+	Alias            graphql.String
+	Contacts         []Contact
+	Id               graphql.ID
+	Manager          User
```

Comment:
Added Alias (since it was not there and i needed it)
and then alphabetized the fields